lightning/src/util/config.rs

```diff
@@ -149,7 +148,22 @@ pub struct ChannelHandshakeConfig {
 	///                as 1000 sats instead, which is a safe implementation-specific lower bound.
 	/// Maximum value: 1,000,000, any values larger than 1 Million will be treated as 1 Million (or 100%)
 	///                instead, although channel negotiations will fail in that case.
-	pub their_channel_reserve_proportional_millionths: u32
+	pub their_channel_reserve_proportional_millionths: u32,
+	/// If set, we attempt to negotiate the `anchors_zero_fee_htlc_tx`option for outbound channels.
+	///
+	/// If this option is set, channels may be created that will not be readable by LDK versions
+	/// prior to 0.0.110, causing [`ChannelManager`]'s read method to return a
+	/// [`DecodeError::InvalidValue`].
+	///
+	/// Note that setting this to true does *not* prevent us from opening channels with
+	/// counterparties that do not support the `anchors_zero_fee_htlc_tx` option; we will simply
+	/// fall back to a `static_remote_key` channel.
+	///
```

Comment:
I think we could add an implementation disclaimer we do not aim to support simple `option_anchor_outputs` and we can add historical context on the rational: https://lists.linuxfoundation.org/pipermail/lightning-dev/2020-September/002796.html